activities/Measure.activity/js/activity.js

```diff
@@ -560,7 +563,7 @@ var app = new Vue({
 					
 					metadata = {
 						mimetype: 'application/pdf',
-						title: "Measure.pdf",
+						title: "Measure Logging by " + vm.currentenv.user.name + ".pdf",
```

Comment:
Should be localized.
Plus, use a parameter {{name}} because name position could depend of the language.